vllm/entrypoints/openai/protocol.py

```diff
@@ -159,7 +159,12 @@ class ChatCompletionRequest(OpenAIBaseModel):
     logit_bias: Optional[Dict[str, float]] = None
     logprobs: Optional[bool] = False
     top_logprobs: Optional[int] = 0
-    max_tokens: Optional[int] = None
+    # TODO: remove max_tokens when field is removed from OpenAI API
```

Comment:
It would be best to tie these todo to an open issue, so all of them can be easily searched. As in "TODO (#XXXX): ..."

Reply:
Good idea!

I will open an issue and update all the TODO comments.

Reply:
@mgoin issue https://github.com/vllm-project/vllm/issues/9845 opened and TODO comments bound to it. Thanks for the suggestion ! :+1: